id/arsip.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="id">
+ 	<head>
+		<title>Indonesia | Indonesia Fansub Database</title>
```

Comment:
Can you rename for all title of theses 3 files "Indonesia Fansub Database" to "The Fansub Database", please?